### PR TITLE
Add bitamp_to_string function

### PR DESCRIPTION
**Pull request by imay**
This CL changes:

1. add function bitmap_to_string, which will convert a bitmap to string
which contains all bit in bitmap
2. add function murmur_hash3_32, which will compute murmur hash for
input strings
3. make the function cast float to string the same with user result
logic

#2706

### CODE REVIEW

File: be/src/exprs/bitmap_function.h [hunk 1]
```diff
@@ -55,6 +55,8 @@ class BitmapFunctions {
     static StringVal bitmap_hash(FunctionContext* ctx, const StringVal& src);
     static StringVal bitmap_or(FunctionContext* ctx, const StringVal& src,const StringVal& dst);
     static StringVal bitmap_and(FunctionContext* ctx, const StringVal& src,const StringVal& dst);
+    static StringVal bitmap_to_string(FunctionContext* ctx, const StringVal& input);
+    static StringVal bitmap_from_string(FunctionContext* ctx, const StringVal& input);
```
Comment (kangkaisen) [inline be/src/exprs/bitmap_function.h]: Would better give a valid string format description.

File: be/test/util/bitmap_test.cpp [hunk 2]
```diff
@@ -267,6 +267,30 @@ TEST_F(BitMapTest, roaring_bitmap_serde) {
...
+TEST_F(BitMapTest, bitmap_to_string) {
+    RoaringBitmap empty;
+    ASSERT_STREQ("", empty.to_string().c_str());
+    empty.update(1);
+    ASSERT_STREQ("1", empty.to_string().c_str());
+    empty.update(2);
+    ASSERT_STREQ("1,2", empty.to_string().c_str());
+}
+
+TEST_F(BitMapTest, bitmap_from_vector) {
```
Comment (kangkaisen) [inline be/test/util/bitmap_test.cpp]: Would better add a invalid string format test case

File: docs/documentation/cn/sql-reference/sql-functions/bitmap-functions/bitmap_from_string.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+-->
+
+# bitmap_from_string
+
+## description
+### Syntax
+
+`BITMAP BITMAP_FROM_STRING(VARCHAR input)`
+
+将一个字符串转化为一个BITAMP，如果输入的字符串不合法，返回NULL.
```
Comment (kangkaisen) [inline docs/documentation/cn/sql-reference/sql-functions/bitmap-functions/bitmap_from_string.md]: Would better describe what's is valid or invalid string input.

File: be/src/exprs/bitmap_function.h (hunk 1 above)
Comment (imay) [reply be/src/exprs/bitmap_function.h]: done

File: docs/documentation/cn/sql-reference/sql-functions/bitmap-functions/bitmap_from_string.md (hunk 3 above)
Comment (imay) [reply docs/documentation/cn/sql-reference/sql-functions/bitmap-functions/bitmap_from_string.md]: done

File: be/test/util/bitmap_test.cpp (hunk 2 above)
Comment (imay) [reply be/test/util/bitmap_test.cpp]: done

**Review by kangkaisen — APPROVED** [review]
+1. LGTM

**Pull request merged**